client/my-sites/pages/page/index.js

```diff
@@ -354,54 +352,6 @@ class Page extends Component {
 		);
 	}
 
-	setComingSoonPage = () => {
-		const pageId = this.props.isComingSoonPage ? 0 : this.props.page.ID;
-
-		this.props.updateSiteFrontPage( this.props.siteId, {
-			wpcom_public_coming_soon_page_id: pageId,
-		} );
-
-		// records whether a page has been set as 'coming soon'
-		// and the paid plan slug
-		this.props.recordTracksEvent( 'calypso_coming_soon_set_page', {
```

Comment:
Did this event ever get registered? Is there such a thing as unregistering an event?

Reply:
I don't see it in the repo, so no it wasn't registered 👍

Reply:
> wpcom_public_coming_soon_page_id

Ah yeah, fair point. Thanks for highlighting it :) I'll remove